src/react/components/common/assetPreview/videoAsset.tsx

```diff
@@ -66,6 +66,15 @@ export class VideoAsset extends React.Component<IVideoAssetProps> {
         duration: 0,
     };
 
+    constructor(props, context) {
+        super(props, context);
+
+        this.moveNextTaggedFrame = this.moveNextTaggedFrame.bind(this);
```

Comment:
this fix make sure `this` is not null.  however, `additionalVideoSetting` is still null

Reply:
Questions for clarification: 
1. Where is `this` null? In each of these functions?
2. Is `additionalSettings` the null value? Or is it `additionalSettings.videoSettings`? It looks like those are coming from the props, which means their value would be set upstream

Reply:
Also, do you have any unit tests to validate the new behavior?

Reply:
1. `this` is null in each of the functions when using keyboard, they're fine when using the on screen button
2. `additionalSettings` is the null value, you can try it out by adding breakpoint.